cmd/main.go

```
@@ -86,26 +86,24 @@ func handleRegistrySetup(g *errgroup.Group, log *zerolog.Logger, cancel context.
+		if err := registry.ReadZotConfig(config.GetZotConfigPath(), &defaultZotConfig); err != nil {
+			log.Error().Err(err).Msg("Error launching default zot registry")
 			return fmt.Errorf("error reading config: %w", err)
 		}
-		defaultZotURL := defaultZotConfig.GetLocalRegistryURL()
-		if err := config.SetRemoteRegistryURL(defaultZotURL); err != nil {
-			log.Error().Err(err).Msg("Error writing the remote registry URL")
-			cancel()
-			return err
-		}
+
+		config.SetRemoteRegistryURL(defaultZotConfig.GetRegistryURL())
```

golangci-lint (1.64.8)

Error return value of `config.SetRemoteRegistryURL` is not checked

(errcheck)